### ISSUE TITLE
Link error?

### ISSUE THREAD

**Issue by rpavlik** [vrpn/vrpn#268]
Tried a clean build on Debian Bullseye here, and I'm now getting missing symbol errors in vrpn_Tracker for libusb, in things that are supposed to be client. (?!)

Very strange...

```
ninja: Entering directory `build'
[7/85] Linking CXX executable client_src/ff_client
FAILED: client_src/ff_client 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/ff_client.dir/ff_client.C.o -o client_src/ff_client  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[8/85] Linking CXX executable client_src/tracker_to_poser
FAILED: client_src/tracker_to_poser 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/tracker_to_poser.dir/tracker_to_poser.cpp.o -o client_src/tracker_to_poser  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[11/85] Linking CXX executable client_src/printvals
FAILED: client_src/printvals 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/printvals.dir/printvals.C.o -o client_src/printvals  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[12/85] Linking CXX executable client_src/forcedevice_test_client
FAILED: client_src/forcedevice_test_client 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/forcedevice_test_client.dir/forcedevice_test_client.cpp.o -o client_src/forcedevice_test_client  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[13/85] Linking CXX executable client_src/phan_client
FAILED: client_src/phan_client 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/phan_client.dir/phan_client.C.o -o client_src/phan_client  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[16/85] Linking CXX executable client_src/clock_drift_estimator
FAILED: client_src/clock_drift_estimator 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/clock_drift_estimator.dir/clock_drift_estimator.C.o -o client_src/clock_drift_estimator  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[17/85] Linking CXX executable client_src/sphere_client
FAILED: client_src/sphere_client 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/sphere_client.dir/sphere_client.C.o -o client_src/sphere_client  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[18/85] Linking CXX executable client_src/c_interface_example
FAILED: client_src/c_interface_example 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/c_interface_example.dir/c_interface_example.c.o client_src/CMakeFiles/c_interface_example.dir/c_interface.cpp.o -o client_src/c_interface_example  libvrpn.a  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[21/85] Linking CXX executable client_src/vrpn_orientation
FAILED: client_src/vrpn_orientation 
: && /usr/bin/c++ -std=c++17 -fPIC -g -rdynamic client_src/CMakeFiles/vrpn_orientation.dir/vrpn_orientation.C.o -o client_src/vrpn_orientation  libvrpn.a  -lglut  -lXmu  -lXi  -lGL  -lGLU  quat/libquat.a  -lm  -lpthread && :
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::vrpn_Tracker_USB(char const*, vrpn_Connection*, unsigned short, unsigned short, long)':
/home/ryan/src/vrpn/vrpn_Tracker.C:1110: undefined reference to `libusb_init'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1118: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1128: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1133: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1135: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::~vrpn_Tracker_USB()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1149: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1153: undefined reference to `libusb_exit'
/usr/bin/ld: libvrpn.a(vrpn_Tracker.C.o): in function `vrpn_Tracker_USB::mainloop()':
/home/ryan/src/vrpn/vrpn_Tracker.C:1241: undefined reference to `libusb_close'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1244: undefined reference to `libusb_open_device_with_vid_pid'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1252: undefined reference to `libusb_claim_interface'
/usr/bin/ld: /home/ryan/src/vrpn/vrpn_Tracker.C:1255: undefined reference to `libusb_close'
collect2: error: ld returned 1 exit status
[24/85] Building CXX object client_src...int_devices.dir/vrpn_print_devices.C.o
````

I mean, I can add a libusb entry to the link, but I feel like there's probably something going wrong here on some other level that these apparently client apps are trying to use libusb at all...

**Comment by rpavlik** [comment]
TBH I could probably go through and clean up/overhaul the build system again sometime to make it modern again. I'm just really low on R&D time at the moment - lot of client projects - so haven't had a chance to get to any of these "want to" projects, only the "have to" (can you tell I have young children ;D )

**Comment by russell-taylor** [comment]
Thanks for tracking it down! Fixed in 777a6872e25ba42e9ac7e4b083302849861b6259

I don't have the young children issue, so should have been paying more attention. :-)
